indico/modules/categories/templates/management/move_categories.html

```diff
@@ -1,18 +0,0 @@
-{% from 'confirmation_dialog.html' import confirmation_dialog %}
```

Comment:
I discussed with @pferreir and we concluded that showing how many are getting moved is probably enough.